logistics/scripts/modules/database/database-lifecycle.psm1

```diff
@@ -142,45 +142,47 @@ function Initialize-EdFiDatabaseWithDbDeploy {
     Write-InvocationInfo $MyInvocation
 
     if ($engine -eq 'PostgreSQL') {
-        if ($dropDatabase) {
-            $params = @{
-                serverName = $csb.host
-                portNumber = $csb.port
-                userName = $csb.username
-                databaseToRemove = $csb.Database
-            }
-            Remove-PostgreSQLDatabase @params
+        $scriptParams = @{
+            serverName   = $csb.host
+            portNumber   = $csb.port
+            userName     = $csb.username
+            databaseName = $csb.database
         }
+    
+        Remove-PostgresSQLDatabaseAsTemplate @scriptParams
```

Comment:
This looks like it is trying to remove the template database every time you call Initialize-EdFiDatabaseWithDbDeploy. Should we do a check like below to only do this if it is a template?

Reply:
done